example_camayoc.yaml

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Leaving this comment as a reminder to rename to `example_config.yaml` if that is the name chosen to be used.